plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -72,6 +74,17 @@
      */
     private WriteConsistencyLevel consistency = WriteConsistencyLevel.DEFAULT;
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
+     * rejects is about 25 seconds per rejection.
+     */
+    private TimeValue retryBackoffInitialTime = timeValueMillis(50);
+
+    /**
+     * Total number of retries attempted for rejections. Defaults to 10.
+     */
+    private int maxRetries = 10;
```

Comment:
I'm not sure that negative numbers actually break vint. I think its just a poor encoding for them. Either way, I'll add validation to the request though because a negative retry won't work properly.